docs/.vuepress/components/eslint-code-block.vue

```diff
@@ -90,6 +96,14 @@ export default {
         rules: this.rules,
         parser: 'vue-eslint-parser',
         parserOptions: {
+          parser: this.typescript
+            ? this.tsEslintParser
+            : this.langTs
+            ? {
+                ts: this.tsEslintParser,
+                typescript: this.tsEslintParser
+              }
+            : null,
```

Comment:
It's really hard to read this. Can we extract part of this to a separate variable to make it more readable? Or use `if`? Or configure Prettier to allow parentheses here?

Also, is the second check really needed? Or can we always set the TS parser for the `ts` and `typescript` blocks?